cmd/notary/main.go

```diff
@@ -37,14 +37,11 @@ func parseConfig() {
 	}
 
 	if trustDir == "" {
-		// Retrieve current user to get home directory
-		usr, err := user.Current()
+		// Get home directory for current user
+		homeDir, err := homedir.Dir()
 		if err != nil {
-			fatalf("cannot get current user: %v", err)
+			fatalf("cannot get current user home directory: %v", err)
 		}
-
-		// Get home directory for current user
-		homeDir := usr.HomeDir
 		if homeDir == "" {
```

Comment:
I left this in deliberately.  It is possible that go-homedir might (at some point in the future if not now) return no error and a blank string.  That could lead to the path being based, silently, on the current dir instead of the home dir.

I'm happy to remove it but it felt more defensive this way.  Let me know.